services/net/filemonitor/FilemonitorAction.cs

```diff
@@ -208,7 +208,7 @@ private async Task CopyFileAsync(SftpClient client, IngestModel ingest, string p
     /// <returns></returns>
     protected string GetOutputPath(IngestModel ingest)
     {
-        return this.Options.VolumePath.CombineWith(ingest.DestinationConnection?.GetConfigurationValue("path")?.MakeRelativePath() ?? "", $"{ingest.Source?.Code}/{GetDateTimeForTimeZone(ingest):yyyy-MM-dd/}");
+        return this.Options.VolumePath.CombineWith(ingest.DestinationConnection?.GetConfigurationValue("path")?.MakeRelativePath() ?? "", $"{ingest.Source?.Code}/{GetDateTimeForTimeZone(ingest):yyyy-MM-dd}/");
```

Comment:
Formatting of the date was resulting in an addition `-` character at the end of the date.